.github/workflows/test-cpp-linter.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: cpp-linter/cpp-linter-action@"${{ github.event.inputs.branch_name }}"
+      - uses: cpp-linter/cpp-linter-action@${{ github.event.inputs.branch_name }}
```

Comment:
I vaguely remember trying this in #10 development, but it didn't work for me. I don't think the `uses` option accepts/allows dynamic variables.

Reply:
En, it looks like `users` option doesn't allowed dynamic variables